_docs/_latest/setup/migrate.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrate from Python Installer to Terraform Module
+order: 007
```

Comment:
Is this parameter intended to correspond to the order of elements on the index?

Reply:
Yes!